status/status.go

```diff
@@ -216,3 +223,18 @@ func GenerateSummary(state intgteststat.IntegrationTestStatus, snapshotName, sce
 
 	return summary, nil
 }
+
+// GenerateURLrefferenceToConsole returns string link to integration test pipelinerun logs for tenant
+func GenerateURLrefferenceToConsole(application string, pipelinerun string, namespace string) (string, error) {
+	console_url := os.Getenv("CONSOLE_URL")
+	if console_url == "" {
+		return "", fmt.Errorf("Environment variable CONSOLE_URL is not present within integration-service.")
```

Comment:
Thinking if we want to return an error here upon a missing console URL envvar, or should we just return something that gives the user an idea that it's not an actual URL, but gives them enough to go on?

Reply:
Agreed, lets return something that will let user know that its not actual URL.

Reply:
Adjusted, let me know if that fulfills your requirements.

Reply:
Hm, maybe we can use something that communicates to users that we're missing the console URL, like `CONSOLE_URL_NOT_AVAILABLE`?